spec.bs

```diff
@@ -3487,9 +3494,9 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 
         <dt>"`executionMode`"
         <dd>
-        1. If |value| is "`compatibility`", "`frozen-context`", or "`group-by-origin`",
-          set |ig|'s [=interest group/execution mode=] to |value|.
-        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+        1. If |value| is "`compatibility`", "`frozen-context`", or "`group-by-origin`", set |ig|'s
+          [=interest group/execution mode=] to |value|.
+        1. Otherwise, set |ig|'s [=interest group/execution mode=] to "`compatibility`".

```

Comment:
Note for reviewer:
the implementation does not abort the update, but just set execution mode to "compatibility".